smartsim/_core/mli/infrastructure/control/workermanager.py

```diff
@@ -28,10 +28,12 @@
 import typing as t
 
 import numpy as np
+from dragon.fli import FLInterface
```

Comment:
We want to avoid having a dragon dependency loaded here so we can test worker manager features independent of if dragon is installed. Consider a test that only needs to verify a message is mapped from `capnproto.Request` to the `InferenceRequest` - it cannot be run without having dragon installed.

Review `DragonCommChannel` for an example of wrapping and injecting a dragon implementation of a feature. Notice the separate module containing that.

**WARNING**: the import `from smartsim._core.mli.comm.channel.dragonchannel import DragonCommChannel` is also bad form (my fault). It should be removed also, though this is not part of your issue.

Reply:
Do you think in this case it might be better to do `FLInterface.attach` in the `EnvironmentConfigLoader` then? So that `FLInterface | None` is return with `get_queue()`? Or would that not make a difference?

Reply:
I `FLInterface.attach` in `EnvironmentConfigLoader ` now and `get_queue()` returns `FLInterface | None`, which should solve this problem.